src/client/refresh/soft/sw_polyset.c

```diff
@@ -299,6 +291,41 @@ quotient must fit in 32 bits.
 FIXME: GET RID OF THIS! (FloorDivMod)
 ====================
 */
+#if defined(__i386__) || defined(__amd64__)
+// just exclude ARM32 with FPU, e.g. Pi 1
+static void
+FloorDivMod(int numer, int denom, int *quo, int *rem)
+{
+	int q, r;
+
+	q = numer / denom;
+	r = numer - q * denom;
+	if (-1/2 || 1/-2 || -1/-2) {
+		// long live C89
+		if (r < 0 && r < denom) {
+			q += 1;
+			r -= denom;
+		}
+	} else {
+		// C99 always truncates to 0
+		if ((numer ^ denom) < 0 && r != 0) {
+			q -= 1;
+			r += denom;
+		}
+	}
+	assert(r == numer - q * denom);
```

Comment:
Next code looks like just for testing code until code is developed. I suppose it could be removed if everything works as expected.